docs/repo-docs/reference/boundaries.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: boundaries
+description: API reference for the `turbo boundaries` command
+---
+
+import { ExperimentalBadge } from '#/components/experimental-badge';
+import { Callout } from '#/components/callout';
+
+<ExperimentalBadge>Experimental</ExperimentalBadge>
+
+Boundaries ensure that caching and other Turborepo features work correctly by checking for package manager Workspace violations.
+
+```bash title="Terminal"
+turbo boundaries
+```
+
+This command will notify for three types of violations:
+
+- Importing a file outside of the package's directory
+- Importing a type without declaring the import as a type
```

Comment:
Ehh this is not quite true. We can't detect this in every case. We just check if you're importing from a type declaration package. But that's sort of a lame lint on its own lol